src/StatusQ/Components/StatusListItem.qml

```diff
@@ -83,6 +86,9 @@ Rectangle {
 
     property list<Item> components
 
+    signal itemClicked(string itemId)
```

Comment:
If your goal is to receive an `onClicked` signal, you can rely on the exposed `sensor` property. You can then propagate `itemId` once its moved to `StatusListItem` in `StatusSearchPopup`.